tests/integration/api/test_api.py

```diff
@@ -555,6 +555,32 @@ def test_monitor_can_delete(self):
             "deleted_monitor_id": monitor["id"]
         }
 
+    def test_monitor_can_delete_with_force(self):
+        # Create a monitor.
+        query = "avg(last_1h):sum:system.net.bytes_rcvd{host:host0} > 100"
+        options = {
+            "silenced": {"*": int(time.time()) + 60 * 60},
+            "notify_no_data": False,
+        }
+        monitor = dog.Monitor.create(type="metric alert", query=query, options=options)
+        monitor_ids = [monitor["id"]]
+
+        # Create a monitor-based SLO.
+        name = "test SLO {}".format(time.time())
+        thresholds = [{"timeframe": "7d", "target": 90}]
+        slo = dog.ServiceLevelObjective.create(
+            type="monitor",
+            monitor_ids=monitor_ids,
+            thresholds=thresholds,
+            name=name,
+        )["data"][0]
+
+        # Check if you can delete the monitor with force option
+        options = {"force": True}
```

Comment:
LGTM. We'll also have to add a note about this `options` argument in the docs since it currently mentions it dors not take any JSON args: https://docs.datadoghq.com/api/?lang=python#delete-a-monitor once it's live.